sql/src/main/java/io/crate/user/UserActions.java

```diff
@@ -40,14 +40,17 @@ private UserActions() {
 
     }
 
-    public static SecureHash generateSecureHash(CreateUserAnalyzedStatement statement, Row parameters) throws GeneralSecurityException {
-        SecureString pw = getUserPasswordProperty(statement.properties(), parameters);
-        SecureHash secureHash = null;
-        if (pw != null) {
-            secureHash = SecureHash.of(pw);
-            pw.close();
+    public static SecureHash generateSecureHash(CreateUserAnalyzedStatement statement, Row parameters) throws GeneralSecurityException, IllegalArgumentException {
+        try(SecureString pw = getUserPasswordProperty(statement.properties(), parameters)) {
```

Comment:
space missing after try. Wonder if checkstyle catches that?

Reply:
It did ;)

```
[ant:checkstyle] [ERROR] /var/lib/jenkins/workspace/CrateDB/sql_tests_on_PR/sql/src/main/java/io/crate/user/UserActions.java:44:12: WhitespaceAround: 'try' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3) [WhitespaceAround]
```

Reply:
fixed it on the feature branch ;)